definitions/variable/macro-economy/tag_econ_sectors.yaml

```
@@ -0,0 +1,10 @@
+- Econ sectors:
```

Please remove "the" in the description such that the concatenated text is readable and capitalize the first word of the description.

Also, please remove the tier in the tag-list, see also my other corresponding comment with the "Value Added" variable.
```suggestion
- Economic sectors:
```